terraform/install.sh

```diff
@@ -198,6 +198,8 @@ applyTerraform() {
 authenticateCluster() {
   CLUSTER_ZONE=$(gcloud container clusters list --filter="name:cloud-ops-sandbox" --project $project_id --format="value(zone)")
   gcloud container clusters get-credentials cloud-ops-sandbox --zone "$CLUSTER_ZONE"
+  # Make alias for this kubectl context
+  kubectx main=.
```

Comment:
why do we need this? We don't bundle kubectx into our custom image and I think it's unnecessary dependency

Reply:
We need kubectl context to match the cluster (the hipstershop and loadgen are on different clusters). `kubectx` is a pretty easy way to do it (and aliasing it for the user makes a better user exp in my opinion) and @Daniel-Sanche told me the dependency is included in the cloud shell

Reply:
Yeah, kubectx is baked into the default Cloud Shell image. I think we should make use of it so we can switch between clusters without needing to find the zone each time

Reply:
Surprised it's included in base cloud shell image, but ok.